modules/RTC/RTCUtils.js

```diff
@@ -779,10 +360,17 @@ class RTCUtils extends Listenable {
             };
             this.getTrackID = ({ id }) => id;
         } else {
-            const message = 'Endpoint does not appear to be WebRTC-capable';
+            this.RTCPeerConnectionType = RTCPeerConnection;
```

Comment:
Do we want to add a check here for browser.isSupported() ? Don't we want to throw the endpoint not webrtc-capable error if its not supported ?

Reply:
I don't think so. We have a checked in JitsiMeetJS already: isWebRtcSupported, which calls that. I'd say that needs to be called before calling JitsiMeetJS.init. Or check can check within, but not this deep.